provider/resource.go

```diff
@@ -819,7 +819,8 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_eventhub_cluster": {Tok: azureResource(azureEventHub, "Cluster")},
 
 			// Eventgrid
-			"azurerm_eventgrid_system_topic": {Tok: azureResource(azureEventGrid, "getSystemTopic")},
```

Comment:
this is an expected rename that's "breaking" - we had mislabelled a resource as a datasource